docs/src/homepage/change_history/index.md

```
@@ -0,0 +1,12 @@
+# Changelog History 
+
+![wrench](/homepage/wrench.png) 
+
+to be done...
+
+___ 
+
+### Beispiel: Projekterstellung <Badge type="info" text="December 24" />
```

Markdownlint (0.35.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)